### ISSUE TITLE
Martin Santos is Nutty's name

### ISSUE THREAD

**Issue by CharlesShares** [NXFinity/ONEORG#1]
view title. I don't like that guy. Get him. And unblock me on Twitter. I can give you more. Also he lives in Australia.

**Comment by NXFinity** [comment]
Name on Twitter and we will unblock

**Comment by CharlesShares** [comment]
No, talking here is fine. I didn't want to get involved but he is an arrogant prick and was being a bully. Your tactic of the  (chat gpt generated) letter worked, you scared him, he backed off and blocked you since he saw he's not invincible. A bully learned his lesson.

So you're welcome for the assist. :-) Thank you for not fully doxxing him.

I would just leave it alone at this point. You can make a note stating you had the last laugh, but just move on. If you wanna scare/hurt him more, look at his sponsors and send them the tweets he has been sending over the past few days. 

Don't just @ them on Twitter, go to the sponsors website and use the contact emails. And when you email them, see how their email address system works, look up the name of all the higher-ups at that company and write to their board / CEO. (Companies get scared when you are a brand risk and even if they don't see it the same way you do, they still might cut him or at least he will be on thin ice.)

for example if their email is customer.response@corsair.com the CEO and everyone on that board is probably "john.smith@corsair.com" if that makes sense. 

He went at you for FOUR days for giggles. This is just in case you want to fire back. You won't win on Twitter any more than you just have. Personally I would take the win and walk away. But there's the option.

**Comment by NXFinity** [comment]
Thanks for the name, it helped a lot.

Yes the one on Twitter is a GPT template, the official one we've not been stupid enough to reveal, as it's going out to several who very publicly have their email addresses on Twitter, which is easy enough to get contact details for from Gamil. Martin's was the one we were struggling the most with, but we have already been in touch with Elgato and a few others about his behaviour.

As with any litigation against multiple individuals, we've documented everything, we are in law and so it is a poignant task to always document everything, no matter how big or small. The irony of it all, is all our details will be on our website in a few days, it's only reason newest update isn't live, as we are waiting upon two of our registration numbers.

The moment he blocked, was the moment for us, it was the point of 'he knew he'd fucked up' and was hoping for us to block him as a 'win', so it was easier just to let him and others keep digging. We knew eventually he would say the wrong thing and then spiral his career in to an oblivion. For someone who claims he helps and supports mental health..

The other irony in all this, is coming at an account, that not only is a project/development and so no real hassle of dealing with muppets, our engagement went through the roof (wrong reasons, but the storm will calm).
